composer.json

```diff
@@ -4,20 +4,26 @@
     "autoload": {
         "psr-4": {
             "Webfactory\\Bundle\\NavigationBundle\\": ""
-        }
+        },
+        "exclude-from-classmap": [
```

Comment:
Das scheint mir hier falsch.

Reply:
Möglich! Ich kam drauf, weil der `composer-require-checker` Klassen aus `vendor` durchsucht und bemängelt hat. Ich vermute, wir haben das Problem sonst nicht, weil wir sonst eher so etwas haben wie

```
"psr-4": {
            "Webfactory\\Bundle\\NavigationBundle\\": "src"
        }
},
```

Vorschlag: alles in `src/` verschieben und Autoloading anpassen?

Reply:
Vorschlag selbst für gut befunden und umgesetzt ^^